src/test/java/com/dragonballs/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.dragonballs.controllers;
+
+import com.dragonballs.entities.User;
+import com.dragonballs.responsedatamapping.UserLocationCreator;
+import com.dragonballs.services.user.UserService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.ResponseEntity;
+
+import javax.jws.soap.SOAPBinding;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserControllerTest {
+
+    @InjectMocks
+    private UserController userController;
+
+    @Mock
+    private UserService userService;
+
+    @Mock
+    private UserLocationCreator userLocationCreator;
+
+    @Before
+    public void setUp(){
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void createResponse_ifUserProvided(){
+        User user = new User();
+        Mockito.when(userService.registerUser(user)).thenReturn(user);
```

Comment:
You can 
import static org.mockito.Mockito.when;
and import static [can't remember :D ]Assert.assertEquals;

so you could call when and assertEquals directly